Lib/multiprocessing/synchronize.py

```diff
@@ -59,6 +60,10 @@ def __init__(self, kind, value, maxvalue, *, ctx):
                     unlink_now)
             except FileExistsError:
                 pass
+            except OSError as e:
+                if e.errno == errno.ENOSPC:
+                    raise Exception("There is not enough space in the"
+                                    " file system(tmpfs) to create a semaphore") from e
```

Comment:
What does "tmpfs" refer to here? Is it posix specific?

Reply:
I agree with Adam here, since this path runs on Windows as well, mentioning tmpfs is misleading.